spec/easy_tags/easy_tags_on_spec.rb

```diff
@@ -91,8 +91,8 @@ class SubclassTaggableModel < TaggableModel; end
         end
       end
 
-      it 'does not raise error ' do
-        is_expected.to raise_error('invalid options')
+      it 'raises an error' do
+        expect(&subject).to raise_error('invalid options')
```

Comment:
Fixes rspec deprecation:

```
  1) easy_tags_on options processing when passed invalid options does not raise error
     Failure/Error: is_expected.to raise_error('invalid options')
       The implicit block expectation syntax is deprecated, you should pass a block rather than an argument to `expect` to use the provided block expectation matcher or the matcher must implement `supports_value_expectations?`. e.g  `expect { value }.to raise Exception with "invalid options"` not `expect(value).to raise Exception with "invalid options"`
     # ./spec/easy_tags/easy_tags_on_spec.rb:95:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:35:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:34:in `block (2 levels) in <top (required)>'
```